pulp_ansible/app/serializers.py

```diff
@@ -963,6 +963,38 @@ class Meta(CollectionImportListSerializer.Meta):
         fields = CollectionImportListSerializer.Meta.fields + ("error", "messages")
 
 
+class CollectionVersionCopyMoveSerializer(serializers.Serializer):
+    """
+    Copy or move collections from a source repository into one or mor destinations.
```

Comment:
```suggestion
    Copy or move collections from a source repository into one or more destinations. This will carry associated content like Signatures and Marks along.
```